src/Label/ReadModels/JSON/Repository/AppConfigReadRepositoryDecorator.php

```diff
@@ -48,7 +48,7 @@ public function canUseLabel(string $userId, string $name): bool
         }
 
         $labels = $config['labels'] ?? [];
-        return in_array($name, $labels);
+        return in_array(strtolower($name), array_map('strtolower', $labels));
```

Comment:
I think it's fine to convert the input array to lowercase for now. But in the future, this can become a waste of CPU power. Instead, make sure that the labels inside the config file are always lowercase.